dll/win32/shell32/lang/fr-FR.rc

```diff
@@ -48,11 +48,11 @@ BEGIN
     MENUITEM SEPARATOR
     MENUITEM "C&ouper\tCtrl+X", FCIDM_SHVIEW_CUT
     MENUITEM "&Copier\tCtrl+C", FCIDM_SHVIEW_COPY
-    MENUITEM "Co&ller\tCtrl+V", FCIDM_SHVIEW_INSERT
-    MENUITEM "Créer un &lien", FCIDM_SHVIEW_INSERTLINK
+    MENUITEM "Co&ller\tCtrl+V", FCIDM_SHVIEW_INSERT //FIXME: accelerator collision &L
+    MENUITEM "Créer un &lien", FCIDM_SHVIEW_INSERTLINK //FIXME: accelerator collision &L
```

Comment:
```suggestion
    MENUITEM "Coll&er\tCtrl+V", FCIDM_SHVIEW_INSERT
    MENUITEM "Créer un &lien", FCIDM_SHVIEW_INSERTLINK
```

Reply:
In the next menu below, there's a `MENUITEM "Coller", IDM_INSERT` item. You may also assign an accelerator to the "e" letter.

Reply:
Thank you very much for your accelerator suggestions as a native french speaker with a french Windows at hand. Finally a useful review-comment. I applied all your suggestions in the accelerator context.